[hotfix/sonarqube] error when running ./gradlew build sonar --info

```
Run ./gradlew build sonar --info
/home/runner/work/_temp/929479a4-228e-4d16-abd4-f64363d13e14.sh: line 1: ./gradlew: No such file or directory
Error: Process completed with exit code 127.

--- a/.github/workflows/sonarqube.yml
+++ b/.github/workflows/sonarqube.yml
@@ -1,4 +1,4 @@
-name: Build
+name: sonarqube
 
 on:
   push:
@@ -13,7 +13,9 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Set up JDK 11
+      - name: chdir
+        run: cd backend
+      - name: Set up JDK 17
         uses: actions/setup-java@v1
         with:
           java-version: 17

